app/src/main/java/com/xmartlabs/doapp/ui/SinginFragment.java

```diff
@@ -59,7 +59,8 @@ void onClickedSingIn() {
       return;
     }
     if (user.getPassword().equals(password.getText().toString())) {
-      Intent intent = Henson.with(getContext()).gotoOnBoardingActivity().build();
+//      Intent intent = Henson.with(getContext()).gotoOnBoardingActivity().build();
```

Comment:
The commented one is the real call that it has to be done, I'll let it clear in the code. The uncommented line is the one that will be removed.